src/Apps/ArtQuiz/__tests__/ArtQuizResults.jest.tsx

```diff
@@ -0,0 +1,53 @@
+import { ArtQuizResults } from "Apps/ArtQuiz/ArtQuizResults"
+import { render, screen } from "@testing-library/react"
+
+const translationFileMock = {
```

Comment:
We *really* shouldn't have to do this — there's no way to load this automatically? This is wild.

Reply:
oooooh wait, no need to do that anymore! Maybe my fault for not communicating the change better but the setupTestWrapper files use these two, one for [enzyme](https://github.com/artsy/force/blob/main/src/DevTools/setupTestWrapper.tsx#L16) one for [rtl](https://github.com/artsy/force/blob/main/src/DevTools/setupTestWrapper.tsx#L20)

Reply:
It was my intention to complete this PR and then lift out into some sort of a global mock for i18n. 

Do we really want to wrap every test for a translated component in the full test wrapper? It seems like we should be able to test simple components in relative isolation.

Reply:
You could just use the re-exported `render` as @gkartalis points out.

Reply:
Agreed. Just looked at the file and saw that those are separate exports. 

@gkartalis did you try to create a global default? Did that implementation fall short somehow?

Reply:
I went with the simplest solution at that point that I found, which was to re-export the testing functions wrapped, but happy to go with any other solution if you have something better in mind 🙂